mentoring/how_to_give_great_feedback.md

```diff
@@ -65,23 +65,22 @@ end
 b = a ? 1 : 2
 ```
 
-You could just say "Write `b = a ? 1 : 2` instead of the `if` statement on L10."
-But that wouldn't be helping a student understand the why or have a lightbulb moment of their own.
+You could simply say, "Write `b = a ? 1 : 2` instead of the `if` statement on L10," but that wouldn't be helping a student understand the _why_ or have a lightbulb moment of their own.
 
-Instead it would be much better to introduce the idea of a ternary, and pose the student a challenge.
+Instead it would be much better to introduce the idea of a ternary and pose the student a challenge.
```

Comment:
```suggestion
Instead it would be much better to introduce the idea of a ternary, posing a challenge to the student.
```

Reply:
This suggested change would transform the final clause into the passive voice which most English-language styles guides discourage. I don't have a linguistic style guide for the site I can reference. Perhaps we could get a confirming opinion to accept this change?

Reply:
I'm fine with either. Your call.

Reply:
The way that I took the sentence here is that we would introduce the idea of a ternary and then pose a challenge to the student, which would be another action after the introduction.  I think introducing the ternary would pose a challenge to the student in the same action, so the comma and change of wording may communicate this.  Not passive, the action is proposed, the effect presented.